Example/src/App.tsx

```diff
@@ -72,104 +84,144 @@ if (Platform.OS === 'android') {
 type Screens = Record<string, { screen: React.ComponentType; title?: string }>;
 
 const SCREENS: Screens = {
+  CardExample: {
+    screen: CardExample,
+    title: '[SET] Card',
+  },
+  CustomTransitionExample: {
+    screen: CustomTransitionExample,
+    title: '[SET] Custom transition',
+  },
+  GalleryExample: {
+    screen: GalleryExample,
+    title: '[SET] Gallery',
+  },
+  LayoutAnimationExample: {
+    screen: LayoutAnimationExample,
+    title: '[SET] Layout Animation',
+  },
+  ManyScreensExample: {
+    screen: ManyScreensExample,
+    title: '[SET] Many screens',
+  },
+  ManyTagsExample: {
+    screen: ManyTagsExample,
+    title: '[SET] Many tags',
+  },
+  NestedStacksExample: {
+    screen: NestedStacksExample,
+    title: '[SET] Nested stacks',
+  },
+  ModalsExample: {
+    screen: ModalsExample,
+    title: '[SET] Modals',
+  },
+  FlatListExample: {
+    screen: FlatListExample,
+    title: '[SET] FlatList',
+  },
+  RestoreStateExample: {
+    screen: RestoreStateExample,
+    title: '[SET] Restore State',
+  },
   ColorInterpolation: {
     screen: ColorInterpolationExample,
     title: 'Color interpolation',
   },
   DeleteAncestorOfExiting: {
     screen: DeleteAncestorOfExiting,
-    title: '🆕 Deleting view with an exiting animation',
+    title: 'Deleting view with an exiting animation',
```

Comment:
Can we mark all existing examples that use Layout Animations with `[LA]` prefix as well?